melosys-eessi-app/src/test/java/no/nav/melosys/eessi/identifisering/BehandleBucIdentifisertServiceTest.java

```diff
@@ -97,13 +97,43 @@ void bucIdentifisert_3SEDer_oppretter2JournalposterOgPubliserer3Meldinger() {
         assertThat(sed3.getJournalpostId()).isEqualTo(JOURNALPOST_ID_3);
     }
 
+    @Test
+    void bucIdentifisert_SEDErX100_ignoreres() {
+        // Første SED som må identifiseres journalføres sammen med opprettelse av oppgave til ID fordeling
+        var sedAlleredeJournalført = lagSedMottattHendelse("1", RINA_SAKSNUMMER, RINA_DOKUMENT_ID, JOURNALPOST_ID, false);
+        // Den påfølgende SEDen blir ignorert siden den er X100
```

Comment:
Unødvendig kommentar: Dette her beskrives jo i metodenavnet

Reply:
Kopierte den metoden ovenfor - så dette er egentlig mest for å klargjøre forskjellen på de to metodene.

Reply:
Kan ev. kalle sed2 til noe mer beskrivende i stedet

Reply:
[19ca913](https://github.com/navikt/melosys-eessi/pull/481/commits/19ca9132a4ea13be48d1abe4130b1697093080e8) ?